Thanks for merge files

mcmgj opened the pull request:
as previously wrote, files are updated 👍 
please verify dashboard.html googlemap key if ok ... 

this last version is verifyed by some users on local PC ... 

with the last VR change, new messages are added, so I want push this version before treat these messages 

thanks

Guy.

mcmgj committed:
Fix Sort DTF, Add SpeedF and Standing table

Sort DTF with or without parentheses
Add Speed to Finish Line column 
Add Standing form (for other users informations)

mcmgj committed:
Add files via upload

mcmgj committed:
Add files via upload

toxcct commented on dashboard.html:
```diff
@@ -10,7 +10,7 @@
     <link rel="icon" href="zicon.png">
     <link rel="stylesheet" href="style.css">
-    <script src="https://maps.googleapis.com/maps/api/js"></script>
+	<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC6bZY8_KuFSGpU5CybugndmQmbFM4fLuU"></script>
```
What is this key all about ?

toxcct commented on dashboard.html:
```diff
@@ -58,10 +58,14 @@
 	        <input type="radio" name="tabs" id="ts:4" />
-	        <label for="ts:4">Map</label>   
-	      </li-->
+	        <label for="ts:4">Classements</label>   
```
Better use "Rankings" as a label, because "Classement is a french word...

toxcct commented on dashboard.html:
```diff
@@ -77,6 +81,9 @@
 	    <div style="white-space: pre;" id="rawlog"></div>
       </div>
       <div id="tab-content4" class="content hidden">
+	    <div id="classement"></div>
```
Better use "Ranking" as a label, because "classement" is a french word...

mcmgj commented:
Yes, the tag "Classements"/"Ranking" will be disappear .. 
The key also ... 

Thanks 



----- Mail original -----
De: "toxcct" <notifications@github.com>
À: "mak08/VRDashboard" <VRDashboard@noreply.github.com>
Cc: "mcmgj" <guy.juy@free.fr>, "Author" <author@noreply.github.com>
Envoyé: Jeudi 8 Novembre 2018 17:18:37
Objet: Re: [mak08/VRDashboard] Thanks for merge files (#8)



@toxcct commented on this pull request. 


In dashboard.html : > @@ -10,7 +10,7 @@
     <title>VR Dashboard</title>
     <link rel="icon" href="zicon.png">
     <link rel="stylesheet" href="style.css">
-    <script src="https://maps.googleapis.com/maps/api/js"></script>
+	<script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC6bZY8_KuFSGpU5CybugndmQmbFM4fLuU"></script> 

What is this key all about ? 

— 
You are receiving this because you authored the thread. 
Reply to this email directly, view it on GitHub , or mute the thread .

mcmgj committed:
Add files via upload

  - suppress error if pos is not in message
  - correct dtf sort
  - correct colspan=15

mcmgj committed:
Add files via upload

mcmgj commented:
- suppress error if pos is not in message
  - correct dtf sort
  - change colspan="16" by colspan="15"

mcmgj commented:
- suppress error if pos is not in message
  - correct dtf sort
  - change colspan="16" by colspan="15"

two other are not needed, function was changed
